MDGameSynchronizer/MDGameSynchronizer.cs

```diff
@@ -70,6 +70,7 @@ public class MDGameSynchronizer : Node
         protected int NodeCount = -1;
 
         protected bool NodeSynchCompleted = false;
+        private int MaxPing;
```

Comment:
I think like the rest of the variables this should be protected since we allow overwriting the MDGameSynchronizer